exit.go

```diff
@@ -545,6 +554,14 @@ func (te *TunaExit) StartReverse(serviceName string) error {
 				te.readUDP()
 			}
 
+			go func() {
+				var np *nkn.NanoPay
+				for {
+					time.Sleep(DefaultNanoPayUpdateInterval)
+					go sendNanoPayment(np, session, te.Wallet, &te.reverseBytesIn, &te.reverseBytesOut, &te.reverseBytesInPaid, &te.reverseBytesOutPaid, te.Common, "0")
```

Comment:
1. No need to use goroutine here.
2. It's better for `sendNanoPayment` function to return error.
3. It's better to check if `te.closeChan` is closed in the beginning of each iteration, and return if it's closed to avoid goroutine leak.